core/api/controller/transaction.ts

```diff
@@ -0,0 +1,17 @@
+import { messageON } from 'shared/util/bus';
+import { API } from 'core/api/util/decorators';
+import { Message } from 'shared/model/message';
+
+export class TransactionController {
+
+    @API('TRANSACTION_CREATE')
+    public transactionCreate(message: Message): void {
```

Comment:
implemented in transaction controller in core

Reply:
Here is the future validation before add to rxjs in the core